bsp/test/org/jetbrains/bsp/project/importing/BspResolverLogicTest.scala

```diff
@@ -27,4 +28,38 @@ class BspResolverLogicTest {
     assert(rootModule.data.targets.head == target)
   }
 
+  @Test
+  def testCalculateModuleDescriptionsBaseDirWithTest(): Unit = {
+    val target = new BuildTarget(
+      new BuildTargetIdentifier("ePzqj://jqke:540/n/ius7/jDa/t/z78"),
```

Comment:
I used the same `BuildTargetIdentifier` than the previous test but I don't really know whether it was randomly generated or not.